lib/osv/vulnerability.proto

```diff
@@ -148,14 +134,23 @@ message Affected {
   google.protobuf.Struct database_specific = 5;
 }
 
-// Vulnerability severity information. The exact meaning of these levels are
-// specific to OSS-Fuzz.
-enum Severity {
-  NONE = 0;
-  LOW = 1;
-  MEDIUM = 2;
-  HIGH = 3;
-  CRITICAL = 4;
+message Severity {
+  // Type of the severity.
+  enum Type {
+    UNSPECIFIED = 0;
+    CVSS_V3 = 1;
+  }
+  // The type of this severity entry.
+  Type type = 1;
+  // The quantitative score.
+  string score = 2;
```

Comment:
if it is quantitative score, then shouldn't this be int or float ?

Reply:
This is a string for future extensibility, or if we relax the "quantitative" requirement.